app/controllers/jobs_controller.rb

```diff
@@ -1,12 +1,50 @@
 class JobsController < ApplicationController
   def index
-    @jobs = jobs_payload
+    @jobs = Job.all
+    render json: {jobs: @jobs}
+  end
+
+  def my_jobs
+    @jobs = current_user.jobs + Job.where(provider_id: current_user.id)
+    render json: {jobs: @jobs}
   end
 
   def new
     render 'new.html.erb'
   end
 
+  def create
+    @job = Job.new(
+      requested_time: params[:requested_time],
+      user_id: current_user.id,
+      status: "posted"
+      )
+    @job.save
+
```

Comment:
What happens if `@job.save` returns `false`?

please add code to take care of this.

#### Reference
[rails docs](https://guides.rubyonrails.org/v3.2/action_controller_overview.html)
serach `create`